### PR TITLE
[Feature] Export VisuallyHidden from the library

### PR DESCRIPTION
**Pull request by LJKaski**
## Description
This PR exports VisuallyHidden from the library and offers basic examples of its use.

## Motivation and Context
This component was requested by the developers using the library, and since we already had an internal implementation of it, it is easy to comply.

## How Has This Been Tested?
Tested manually in styleguidist.

## Release notes
-Add VisuallyHidden component

### CODE REVIEW

File: src/core/VisuallyHidden/VisuallyHidden.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+```js
+import { VisuallyHidden, SearchInput } from 'suomifi-ui-components';
+<>
+  <span>There is a hidden element under this text.</span>
+  <VisuallyHidden>
+    This element will be read by screen readers even though it's
+    visually hidden
+  </VisuallyHidden>
+
+  <SearchInput
```
Comment (aappoalander) [inline src/core/VisuallyHidden/VisuallyHidden.md:12]: Could we replace this with TextInput? It will be more simple after the upcoming breaking for SearchInput.

File: .styleguidist/styleguidist.sections.js [hunk 2]
```diff
@@ -77,7 +78,7 @@ module.exports = {
               ]),
             },
             {
-              name: 'Icons',
+              name: 'Icon',
```
Comment (aappoalander) [inline .styleguidist/styleguidist.sections.js:81]: Why this change? It has two separate components under the heading.

File: src/core/VisuallyHidden/VisuallyHidden.md (hunk 1 above)
Comment (LJKaski) [reply src/core/VisuallyHidden/VisuallyHidden.md:12]: Search input was a common example I saw people use online when talking about where VisuallyHidden styles are used. Text input not so much. That's why I opted for this.

File: .styleguidist/styleguidist.sections.js (hunk 2 above)
Comment (LJKaski) [reply .styleguidist/styleguidist.sections.js:81]: It was to match it with the others. For example, Text and Link also contain several components, but they are in singular form in the navigation. This stood out.

File: src/core/VisuallyHidden/VisuallyHidden.md (hunk 1 above)
Comment (aappoalander) [reply src/core/VisuallyHidden/VisuallyHidden.md:12]: Let's keep this as discussed elsewhere. Just have to remember to update when introducing changes to SearchInput.

**Review by aappoalander — APPROVED** [review]
